databricks/koalas/internal.py

```diff
@@ -395,28 +401,29 @@ class _InternalFrame(object):
 
     def __init__(
         self,
-        sdf: spark.DataFrame,
-        index_map: Optional[List[IndexMap]],
+        spark_frame: spark.DataFrame,
+        index_map: Optional[Dict[str, Optional[Tuple[str, ...]]]],
         column_labels: Optional[List[Tuple[str, ...]]] = None,
-        column_scols: Optional[List[spark.Column]] = None,
+        data_spark_columns: Optional[List[spark.Column]] = None,
         column_label_names: Optional[List[str]] = None,
```

Comment:
For `column_labels` and `column_label_names`, I am currently thinking about reusing Pandas' Index directly.

Reply:
If that doesn't work, I plan to rename `column_labels` -> `data_column_labels`, `data_column_label_names` -> `data_column_label_names`

Reply:
If that doesn't work, I plan to rename something like `column_index_labels` and `column_index_names`.